mkdocs.insiders.yml

```diff
@@ -1,6 +1,9 @@
 INHERIT: mkdocs.yml
 plugins:
     meta:
+    privacy:
+        assets_include:
+          - cdn.amplitude.com/*
```

Comment:
sweet, glad you got the bundler thing working!

I think it's acceptable but fwiw if we build without insiders (such as for a local development build), then we might have a CSP issue if we don't allow the Amplitude CDN. Probably not worth fixing, just thought I'd mention it.

Reply:
Yeah I think that's fine for now. We'll be building w/ insiders for the production build for the foreseeable future